packages/shared/src/features/onboarding/steps/FunnelPricing.tsx

```diff
@@ -79,6 +80,26 @@ const PricingSection = ({
   );
 };
 
+const getPricing = (
+  pricing: BaseProductPricingPreview,
+  pricingType: FunnelPricingType,
+): {
+  amount: string;
+  subtitle: string;
+} => {
+  if (pricingType === FunnelPricingType.Monthly) {
+    return {
+      amount: pricing.price.monthly.formatted,
+      subtitle: 'per month',
+    };
+  }
+
+  return {
+    amount: pricing.price.daily.formatted,
+    subtitle: 'per day',
+  };
+};
```

Comment:
Why is this needed on FE, doesn't the API now simply return it?
Or is it that it returns both and FE depending on x needs to render the right one?

Reply:
Ah, the API returns all three options (original value, monthly, daily). We are just now utilizing the available properties and we will use them depending on what will be configured from the Funnel.

Reply:
Also, the Funnel and our payment system kinda have a weird relationship 😅 

If you are curious about the discussions: https://dailydotdev.slack.com/archives/C08J2D9NKPU/p1747668117785479?thread_ts=1747635961.942809&cid=C08J2D9NKPU